data.sql

```diff
@@ -47,3 +47,42 @@ SET owner_id = CASE
                 WHEN name IN ('Angemon', 'Boarmon') THEN 5
                 ELSE NULL
               END;
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+    ('William Tatcher', 45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+    (1, 1),
+    (3, 1),
+    (3, 2),
+    (4, 2);
+
+
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES
+    (1, 1, '2020-05-24'), -- Agumon visited William Tatcher on May 24th, 2020
+    (1, 3, '2020-07-22'), -- Agumon visited Stephanie Mendez on Jul 22th, 2020
+    (2, 4, '2021-02-02'), -- Gabumon visited Jack Harkness on Feb 2nd, 2021
+    (3, 2, '2020-01-05'), -- Pikachu visited Maisy Smith on Jan 5th, 2020
+    (3, 2, '2020-03-08'), -- Pikachu visited Maisy Smith on Mar 8th, 2020
+    (3, 2, '2020-05-14'), -- Pikachu visited Maisy Smith on May 14th, 2020
+    (4, 3, '2021-05-04'), -- Devimon visited Stephanie Mendez on May 4th, 2021
+    (5, 4, '2021-02-24'), -- Charmander visited Jack Harkness on Feb 24th, 2021
+    (6, 2, '2019-12-21'), -- Plantmon visited Maisy Smith on Dec 21st, 2019
+    (6, 1, '2020-08-10'), -- Plantmon visited William Tatcher on Aug 10th, 2020
+    (6, 2, '2021-04-07'), -- Plantmon visited Maisy Smith on Apr 7th, 2021
+    (7, 3, '2019-09-29'), -- Squirtle visited Stephanie Mendez on Sep 29th, 2019
+    (8, 4, '2020-10-03'), -- Angemon visited Jack Harkness on Oct 3rd, 2020
+    (8, 4, '2020-11-04'), -- Angemon visited Jack Harkness on Nov 4th, 2020
+    (9, 2, '2019-01-24'), -- Boarmon visited Maisy Smith on Jan 24th, 2019
+    (9, 2, '2019-05-15'), -- Boarmon visited Maisy Smith on May 15th, 2019
+    (9, 2, '2020-02-27'), -- Boarmon visited Maisy Smith on Feb 27th, 2020
+    (9, 2, '2020-08-03'), -- Boarmon visited Maisy Smith on Aug 3rd, 2020
+    (10, 3, '2020-05-24'), -- Blossom visited Stephanie Mendez on May 24th, 2020
```

Comment:
- [ ] [OPTIONAL] Similar to my previous comment on dynamically retrieving the IDs, you could use this code 👇🏾 

```sql
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Gabumon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-01-05');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-03-08');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-14');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Devimon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2021-05-04');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Charmander'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-12-21');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-08-10');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2021-04-07');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Squirtle'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2019-09-29');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-10-03');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-04');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-01-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-05-15');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-02-27');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
```